privacyideautils/clientutils.py

```diff
@@ -133,26 +135,30 @@ def set_credentials(self, username, password):
             result = res.get("result")
             if result.get("status") is True:
                 self.auth_token = result.get("value", {}).get("token")
+                if self.pi_authorization:
+                    self.headers = {"Authorization": self.auth_token}
+                else:
+                    self.headers = {"PI-Authorization": self.auth_token}
```

Comment:
According to help "Authorization" should be the default, self.pi_authorization should enable PI-Authorization.

Reply:
You are right. Will do.

Reply:
Oh thanks. You did it.